### PR TITLE
Update to 1.3.257 headers

### PR DESCRIPTION
**Pull request by Tony-LunarG**
Also remove VK_NV_cooperative_matrix and VK_QNX_external_memory_screen_buffer from codegen

### CODE REVIEW

**Comment by Tony-LunarG** [comment]
I have no idea why the Qt installs are timing out in CI.  I don't think it's due to these changes

**Comment by juan-lunarg** [comment]
> I have no idea why the Qt installs are timing out in CI. I don't think it's due to these changes

Maybe upgrading our install-qt-action to v3 would fix the issue

https://github.com/jurplel/install-qt-action

File: build-android/android-generate.sh [hunk 1]
```diff
@@ -50,10 +50,10 @@ echo "Using python: $(which $PYTHON_EXECUTABLE)"
 ( cd generated/include; ${PYTHON_EXECUTABLE} ${VT_SCRIPTS}/vlf_makefile_generator.py ../../../layer_factory ${REGISTRY_PATH}/../include)
 
 # apidump
-( cd generated/include; ${PYTHON_EXECUTABLE} ${VT_SCRIPTS}/vt_genvk.py -registry ${REGISTRY} -scripts ${REGISTRY_PATH} api_dump.cpp)
-( cd generated/include; ${PYTHON_EXECUTABLE} ${VT_SCRIPTS}/vt_genvk.py -registry ${REGISTRY} -scripts ${REGISTRY_PATH} api_dump_text.h)
-( cd generated/include; ${PYTHON_EXECUTABLE} ${VT_SCRIPTS}/vt_genvk.py -registry ${REGISTRY} -scripts ${REGISTRY_PATH} api_dump_html.h)
-( cd generated/include; ${PYTHON_EXECUTABLE} ${VT_SCRIPTS}/vt_genvk.py -registry ${REGISTRY} -scripts ${REGISTRY_PATH} api_dump_json.h)
+( cd generated/include; ${PYTHON_EXECUTABLE} ${VT_SCRIPTS}/vt_genvk.py -registry ${REGISTRY} -scripts ${REGISTRY_PATH} api_dump.cpp -removeExtensions VK_NV_cooperative_matrix -removeExtensions VK_QNX_external_memory_screen_buffer)
```
Comment (mikes-lunarg) [inline build-android/android-generate.sh:53]: Do we need a tracking issue to fix these extensions?

**Review by mikes-lunarg — APPROVED** [review]
LGTM

File: build-android/android-generate.sh (hunk 1 above)
Comment (Tony-LunarG) [reply build-android/android-generate.sh:53]: Yes, I plan to file one after this gets settled

**Pull request merged**